Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -160,6 +173,164 @@ YUI.add('ez-subitemlistview', function (Y) {
+                selectedCell.addClass('ez-subitem-hovered-priority-cell');
+            }
+        },
+
+        /**
+         * Show the error icon to inform the user input is not correctly filled.
+         *
+         * @method _displayErrorIcon
+         * @protected
+         * @param {Object} input the DOM node of the input
+         * @param {Object} selectedCell the DOM node of the current selected cell
+         */
+        _displayErrorIcon: function (input, selectedCell) {
+            //input.addClass('ez-subitem-error-input');
+            selectedCell.addClass('ez-subitem-error-priority-cell');
+            selectedCell.removeClass('ez-subitem-selected-priority-cell');
+        },
+
+        /**
+         * Hide the edit icon to inform the user input can be edited.
+         *
+         * @method _hideEditIcon
+         * @protected
+         * @param {Object} e event facade
+         */
+        _hideEditIcon: function (e) {
+            var selectedCell = this.get('container').one('#priority-cell-' + e.target.getAttribute('data-location-id'));
+            if (!this.get('isOnePrioritySelected')) {
+                selectedCell.removeClass('ez-subitem-hovered-priority-cell');
+            }
+        },
+
+        /**
+         * Hide the error icon.
+         *
+         * @method _hideErrorIcon
+         * @protected
+         * @param {Object} input the DOM node of the input
+         * @param {Object} selectedCell the DOM node of the current selected cell
+         */
+        _hideErrorIcon: function (input, selectedCell) {
+            //input.removeClass('ez-subitem-error-input');
+            selectedCell.removeClass('ez-subitem-error-priority-cell');
+            selectedCell.addClass('ez-subitem-selected-priority-cell');
+        },
+
+        /**
+         * Show the 'validate' & 'cancel' buttons which allow to update or not the priority.
+         *
+         * @method _displayPriorityButtons
+         * @protected
+         * @param {Object} e event facade
+         */
+        _displayPriorityButtons: function (e) {
+            var selectedCell = this.get('container').one('#priority-cell-' + e.target.getAttribute('data-location-id'));
+
+            e.preventDefault();
+            if (!this.get('isOnePrioritySelected')) {
+                this._set('isOnePrioritySelected', true);
+                selectedCell.removeClass('ez-subitem-hovered-priority-cell');
+                selectedCell.addClass('ez-subitem-selected-priority-cell');
+                e.target.removeAttribute('readonly');
+            }
+        },
+
+        /**
+         * Hide the 'validate' & 'cancel' buttons which allow to update or not the priority.
+         * Also fill back priority input with the current sub item priority.
+         *
+         * @method _hidePriorityButtons
+         * @protected
+         * @param {Object} e event facade
+         */
+        _hidePriorityButtons: function (e) {
```

Comment:
it does much more than just hiding the buttons. I would name it '_restorePriorityCell' and it would call 2 methods:

1. _hidePriorityButtons
2. _restorePriorityValue